service_catalog/models/request.py

```diff
@@ -26,7 +26,7 @@
 
 
 class Request(RoleManager):
-    fill_in_survey = models.JSONField(default=dict)
+    fill_in_survey = models.JSONField(default=dict, blank=True)
```

Comment:
I don't remember exactly where. maybe when loading var before sending them to Tower, but we did this before and we got some errors